index.html

```diff
@@ -0,0 +1,194 @@
+      <h2 class="second-section-heading">Concert Program</h2>
+      <hr />
+      <div class="program-list">
+        <div class="lecture flex-box">
+          <img class="flex-box-img icons" src="./images/game1.png" alt="" />
+          <h2>Bike Riding</h2>
+          <p>
+            The best ages to teach your child are between four and six years old. It should take 45 minutes.
+          </p>
+        </div>
+
+        <div class="starup-block flex-box">
+          <img class="flex-box-img icons" src="./images/game2.png" alt="" />
+          <h2 class="font-lato">Card Games</h2>
+          <p class="font-lato">
+            It is a rummy style of game in which the aim is to make melds of seven cards of the same value.
+          </p>
+        </div>
+
+        <div class="growth-hacks flex-box">
+          <img class="flex-box-img icons" src="./images/game3.png" alt="" />
+          <h2 class="font-lato">Baloon Stomp</h2>
+          <p class="font-lato">
+            Balloon Stomp is a frenetic game in which players pop the balloons tied around each other's ankles.
+          </p>
+        </div>
+
+        <div class="why Raise Fund flex-box">
+          <img class="flex-box-img icons" src="./images/game4.png" alt="" />
+          <h2 class="font-lato">Rope Pulling</h2>
+          <p class="font-lato">
+            Pulling the rope along with the members of your team against the opposition team each to their side.
+          </p>
+        </div>
+
+        <div class="forum flex-box">
+          <img class="flex-box-img icons" src="./images/game5.png" alt="" />
+          <h2 class="font-lato">Horse Riding</h2>
+          <p class="font-lato">
+            Horse riding or horseback riding, includes the disciplines of riding, driving, and vaulting.
+          </p>
+        </div>
+      </div>
+      <div class="button-container">
+        <button type="submit" class="program-button font-lato">
+          Join CCS Summer Concert
+        </button>
+      </div>
+    </div>
+  </section>
+
+  <div class="section-speakers-heading">
+    <h2 class="font-lato">Featured Speakers</h2>
+    <hr />
+  </div>
+
+  <section class="partners-section" id="partners">
+    <h2 class="partners-heading font-lato"><strong>Partners</strong></h2>
+    <hr />
+    <div class="partners-flex">
+      <div class="partners-list">
+        <img class="partner-image" src="./images/sponsor1.png" alt="" />
+      </div>
+      <div class="partners-list">
+        <img class="partner-image img" src="./images/sponsor2.png" alt="" />
+      </div>
+      <div class="partners-list">
+        <img class="partner-image img" src="./images/sponsor3.png" alt="" />
+      </div>
+      <div class="partners-list">
+        <img class="partner-image img" src="./images/sponsor4.png" alt="" />
+      </div>
+      <div class="partners-list">
+        <img class="partner-image img" src="./images/sponsor5.png" alt="" />
+      </div>
+    </div>
+  </section>
```

Comment:
- [ ] The **spacings** of contents within your _Partner_ section are too wide apart. Kindly adjust the spacing of contents in this section proportionally, as can be seen in the screenshot of the template design below:

**Your Partner section:**
![image](https://user-images.githubusercontent.com/12745474/132957627-f695b5f2-6791-4ec6-8cd5-975e47844cfd.png)


**The original Partner section:**
![image](https://user-images.githubusercontent.com/12745474/132957634-083b8d8b-eb1e-4b82-9f6f-31be51bc292e.png)